corehq/apps/data_cleaning/tests/test_session.py

```diff
@@ -158,6 +158,18 @@ def test_remove_filters(self):
             self.assertEqual(filters[index].prop_id, prop_id)
             self.assertEqual(filters[index].index, index)
 
+    def test_remove_column(self):
+        session = BulkEditSession.new_case_session(self.django_user, self.domain_name, self.case_type)
+        column_to_remove = session.columns.all()[1]  # owner_name
+        self.assertEqual(column_to_remove.prop_id, 'owner_name')
+        session.remove_column(column_to_remove.column_id)
+        columns = session.columns.all()
+        self.assertEqual(len(columns), 5)
```

Comment:
nit: is this assertion testing that the number of `columns` has, or has not changed? Would be nice to see that clarified, or maybe paired with an earlier assertion to make it clear.

Reply:
added to testing todos thanks